opendds_cmake_module/CMakeLists.txt

```diff
@@ -14,7 +14,4 @@ if(BUILD_TESTING)
 endif()
 
 # Make this package available to other ROS-2 packages.
-ament_package()
-
-# Install CMake utilities.
-install(PROGRAMS ${PROJECT_NAME}/${PROJECT_NAME} DESTINATION lib/${PROJECT_NAME})
```

Comment:
Why is the install removed? I'm  guessing that's fine but did you check that the rmw_opendds still works?

Reply:
Yes - I removed it after I had to remove some other code from rmw_opendds